upgrade/upgrade-scripts/common.sh

```diff
@@ -142,7 +142,14 @@ function get_image_path() {
 }
 
 function get_image_version() {
-	basename "$(get_image_path)"
+#	basename "$(get_image_path)"
+  source_version_information
+  echo "$VERSION"
```

Comment:
Rather than modify this function, I think we might be able to remove it entirely.. e.g. by using `source_version_information` within the `verify-jar` script.. I think that's the only place where this function is currently needed, and we can replace that with `source_version_information`, and then use the `VERSION` and `HOTFIX` variables to pass into the upgrade-verify JAR.. What do you think?

Reply:
`get_image_version` is used in the `upgrade` script as well, but it is used identically to `verify-jar` script. Sure, it is a simplification that can improve readability.

Reply:
it's used in `upgrade`, but the ways it's used is dead code..
```
 23 IMAGE_VERSION=$(get_image_version)                                              
 24 [[ -n "$IMAGE_VERSION" ]] || die "failed to determine image version" 
 ```
 while it's used here, `IMAGE_VERSION` is never used.. so I'm pretty sure we can simply remove these two lines and everything would still work.